packages/analytics/shippers/fullstory/src/fullstory_shipper.test.ts

```diff
@@ -108,7 +108,7 @@ describe('FullStoryShipper', () => {
   });
 
   describe('reportEvents', () => {
-    test('calls the API once per event in the array with the properties transformed', () => {
+    test('filters the events by the allow-list and calls the API once per event in the array with the properties transformed', () => {
```

Comment:
Implementing the change suggested in previous comment would allow to split this test to assert both behavior independently.